system-parachains/people-paseo/src/people.rs

```diff
@@ -48,7 +48,7 @@ parameter_types! {
 	pub const GeneralAdminBodyId: BodyId = BodyId::Administration;
 	pub StableAssetLocation: Location = Location::new(
 		1,
-		[Parachain(1000), xcm::latest::Junction::PalletInstance(50), xcm::latest::Junction::GeneralIndex(3)],
+		[Parachain(2034), GeneralIndex(222)]
```

Comment:
let's reused the constants defined in the other file to avoid future inconsistency.

Reply:
Done